app/controllers/middleware_server_group_controller.rb

```diff
@@ -3,12 +3,44 @@ class MiddlewareServerGroupController < ApplicationController
   include Mixins::GenericShowMixin
   include EmsCommon
   include MiddlewareCommonMixin
+  include Mixins::MiddlewareDeploymentsMixin
 
   before_action :check_privileges
   before_action :get_session_data
   after_action :cleanup_action
   after_action :set_session_data
 
+  OPERATIONS = {
+    :middleware_server_group_reload         => {:op  => :reload_middleware_server_group,
+                                                :msg => N_('Reload')
```

Comment:
I'm getting a bit lost (not saying the code above is incorrect). Where is `_()` being applied to the `:msg` strings here?